src/typography/src/Paragraph.js

```diff
@@ -1,30 +1,22 @@
 import React, { forwardRef, memo } from 'react'
 import PropTypes from 'prop-types'
 import Box from 'ui-box'
-import useParagraphStyle from '../../theme/src/hooks/useParagraphStyle'
+import useStyleConfig from '../../hooks/use-style-config'
+
+const emptyObject = {}
 
 const Paragraph = memo(
   forwardRef(function Paragraph(props, ref) {
-    const { color = 'default', marginTop, size = 400, ...restProps } = props
+    const { marginTop, size = 400, ...restProps } = props
 
-    const { marginTop: defaultMarginTop, ...textStyle } = useParagraphStyle(
-      size,
-      color
+    const textStyle = useStyleConfig(
+      'Paragraph',
+      { size },
```

Comment:
Same here – think we need to support theme lookup for `color` and `fontFamily` like we used to do in v5. (We may need to check that inputs still support this as well!)